backend/generator.js

```diff
@@ -3,6 +3,10 @@ var exports = module.exports = {};
 var uuidV4 = require("uuid/v4");
 
 exports.executeScript = function (socket, formData) {
+  if (!isValid(formData)) {
```

Comment:
I agree btw manipulating the validation would only lead to a build error and nothing else. There shouldn't be any security concerns. But if you still want validation on the server's end, could you not duplicate the isValid method?